library/src/main/java/com/github/pwittchen/reactivenetwork/library/network/observing/strategy/PreLollipopNetworkObservingStrategy.java

```diff
@@ -89,7 +89,7 @@ private Disposable unsubscribeInUiThread(final Action unsubscribe) {
               try {
                 unsubscribe.run();
               } catch (Exception e) {
-                e.printStackTrace();
+                onError(LOG_TAG, e);
```

Comment:
There should not be a `LOG_TAG`! This method has the following signature `onError(final String message, final Exception exception)`. Please put there a descriptive message. E.g. `Could not dispose subscription in UI Thread`.